packages/gp2-components/src/organisms/__tests__/OnboardingPageFooter.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import OnboardingPageFooter from '../OnboardingPageFooter';
+
+describe('OnboardingPageFooter', () => {
+  const defaultProps = {
+    previousHref: '',
```

Comment:
I think you don't need to mention continueHref and isContinueEnabled in the defaultProps because you mention both of them in every render.